apps/files/src/components/FilesAppBar.vue

```diff
@@ -127,7 +127,7 @@
               selectedFiles.length < 1 ? $_ocTrashbin_empty() : $_deleteResources_displayDialog()
             "
           >
-            {{ $_ocAppBar_clearTrashbinButtonText }}
+            <translate>Delete</translate>
```

Comment:
I'm a bit worried that having only "Delete" won't be clear enough when no files are selected. Could we keep the separation and use something like "Delete all files"? Or maybe just "Empty" (this is how mac os shows it in the bin)?

Reply:
ah, I didn't realize that the button was clickable even with zero selection

I'll rename it to "Empty" in that case

Reply:
Great, thanks!

Reply:
done